NanaZip.UI.Classic/SevenZip/CPP/7zip/UI/FileManager/ProgressDialog2.cpp

```diff
@@ -609,7 +616,7 @@ void GetTimeString(UInt64 timeValue, wchar_t *s)
   *s++ = ':'; UINT_TO_STR_2(seconds);
   *s = 0;
 }
-
+/**
```

Comment:
Invalid Modification Mark Usage. Read https://github.com/M2Team/NanaZip/blob/main/CONTRIBUTING.md#modifications-for-inherited-7-zip-mainline-source-code for more information.